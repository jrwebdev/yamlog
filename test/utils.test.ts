import { createMockProject } from './utils';

let mockProject: any;

beforeEach(() => {
  mockProject = createMockProject();
  expect.hasAssertions();
});

afterEach(() => {
  mockProject.teardown();
});

it('should write a changelog', async () => {
  await mockProject.setup({
    '0.1.0': {
      metadata: { date: '2017-01-01' },
      feature: ['Feature 1'],
    },
  });
  const changelog = await mockProject.readChangelog();
  expect(changelog).toMatchSnapshot();
});
