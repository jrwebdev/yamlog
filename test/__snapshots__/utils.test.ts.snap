// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should write a changelog 1`] = `
0.1.0:
  metadata:
    date: '2017-01-01'
  feature:
    - Feature 1

`;
